cob_moveit_bringup/launch/move_group.launch

```diff
@@ -54,7 +56,8 @@
     <param name="allow_trajectory_execution" value="$(arg allow_trajectory_execution)"/>
     <param name="max_safe_path_cost" value="$(arg max_safe_path_cost)"/>
     <param name="jiggle_fraction" value="$(arg jiggle_fraction)" />
-
+	<param name="collision_detector" value="IndustrialFCL" />
```

Comment:
I guess people wanting to use this need to install some things....please add the package for `IndustrialFCL` to the package.xml